backend/EasyMeets.Core/EasyMeets.Core.Common/Enums/SlotType.cs

```diff
@@ -2,6 +2,6 @@ namespace EasyMeets.Core.Common.Enums;
 
 public enum SlotType
 {
-    Personal,
-    Team
+    Personal = 1,
+    Team = 2
```

Comment:
Why did you change enum numbering?

Reply:
I changed enum numbering because slot type in table is saved as value 1 and 2 (maybe this error was made during seeding). 
Because of that, there was error during loading data from database. To remove this error and convenient enum mapping, I set the value as it is in db. 
So, I made it so that the enum value matched the values in the database and avoided adding (+1).

Reply:
It would be better to fix seed and db values

Reply:
So, enum value in db should start with 0?

Reply:
Can I fix this bug in separate branch, I think it doesn't refer to this task and so that every will know what happened and load new db after fixing seed?

Reply:
Yes, sure
Could you please create a card for this?

Reply:
Of course. 
So, I commit with wrong (Personal=1) data, so that my page will receive data. And in another brunch I will fix seed and change enum numbering, is it right?